Cargo.toml

```diff
@@ -224,7 +223,6 @@ storage_controller_client = { path = "./storage_controller/client" }
 tenant_size_model = { version = "0.1", path = "./libs/tenant_size_model/" }
 tracing-utils = { version = "0.1", path = "./libs/tracing-utils/" }
 utils = { version = "0.1", path = "./libs/utils/" }
-vm_monitor = { version = "0.1", path = "./libs/vm_monitor/" }
```

Comment:
I just did `cargo remove procfs -p control_plane`

Reply:
Hmm I guess this is fine, it's a workspace member so perhaps that makes it automatically a dependency. rustls-split being removed I do not understand.

I think it might be best if we didn't do this in unrelated PR but reset the Cargo.toml in a separate PR? I am rather surprised `cargo remove` would do this.

Reply:
I dont mind. Will update

Reply:
Manually reverted `Cargo.toml` changes in https://github.com/neondatabase/neon/pull/8587/commits/f27ce011c006cf7419f054c78410dbfabd889ab8